src/Executor/Executor.php

```diff
@@ -97,15 +97,19 @@ public function createRuntime(
         ];
 
         $endpoint = $this->selectEndpoint($projectId, $deploymentId);
-        $response = $this->call($endpoint, self::METHOD_POST, $route, [ 'x-opr-runtime-id' => $runtimeId ], $params, true, $timeout);
+
+        $body = '';
+        $response = $this->call($endpoint, self::METHOD_POST, $route, [ 'x-opr-runtime-id' => $runtimeId ], $params, true, $timeout, callback: function ($chunk) use ($body) {
+            $body .= $chunk;
+        });
 
         $status = $response['headers']['status-code'];
         if ($status >= 400) {
             $message = \is_string($response['body']) ? $response['body'] : $response['body']['message'];
```

Comment:
I dont need to touch that :thinking: i am pretty sure on errors we use the `->json()` method of the response that sets the content-type from stream back to json. not sure what happens then 😅